src/redux/account/reducer.js

```diff
@@ -0,0 +1,37 @@
+import {
+  LOGIN_USER_START,
+  LOGIN_USER_SUCCESS,
+  LOGIN_USER_FAILURE,
+  LOGOUT_SUCCESS,
+} from './actionType';
+
+const initialState = {
+  accessToken: '',
+  error: '',
```

Comment:
pls add some isLogginIn state, so you can handle that in the component. Otherwise the user wont know whats going on if the log in takes too long time (eg on slow network)